Modelica/Mechanics/MultiBody/Examples/Loops/Utilities/GasForce2.mo

```diff
@@ -24,7 +24,8 @@ model GasForce2 "Rough approximation of gas force in a combustion engine's cylin
   SI.Velocity v_rel "Relative piston velocity (<0: compression;  >0: expansion)";
 
 protected
-  constant SI.SpecificHeatCapacity R_air = Modelica.Constants.R/0.0289651159;
+  constant SI.MolarMass MM_gas = 0.0289651159;
```

Comment:
```suggestion
  constant SI.MolarMass MM_gas = 0.0289651159 "Molar mass for dry air; should be changed";
```
This is somewhat complicated. Just having the value for "air" and calling it "gas" without any explanation seems confusing to me.
I see the following possibilities:

- Explain it as proposed.
- Call it "air" as before - but then we get the question of why there's just air in a combustion chamber.
- Call it "gas" without explanation - but using the name "gas" for "air" seems as if we know it is bad, and just try to hide it.

Reply:
@HansOlsson I fully agree with you. To rename to "gas" was just to be complementary with "gasForce". 
To explain the model as purposed, we need the proper molar mass of the gas. This need some recherche. And, of course, the simulation results will change then.
But I agree this is the way we should preferre.

Reply:
I reviewed the model and, actually, `R_air / R_gas` is used just to calculate the gas' density `dens` which itself is not further used. So I tend to remove the variables `R_air` and `dens` completely. The regression results shall not be concerned.

Reply:
> I reviewed the model and, actually, `R_air / R_gas` is used just to calculate the gas' density `dens` which itself is not further used. So I tend to remove the variables `R_air` and `dens` completely. The regression results shall not be concerned.

Yes, this makes sense: Remove R_air and dens.